README.md

```diff
@@ -4,7 +4,6 @@
 
 This repository contains the conceptual documentation for .NET. The [.NET documentation site](https://docs.microsoft.com/dotnet) is built from multiple repositories in addition to this one:
 
-- [Code samples and snippets](https://github.com/dotnet/samples)
```

Comment:
I wish we would have linked to the .NET browser experience here instead of just removing this.

```markdown
[.NET code samples and snippets](https://docs.microsoft.com/samples/browse/?products=dotnet)
```

Reply:
@Youssef1313 Yeah I like @IEvangelist idea. However, we should remove the word `snippets`.

Reply:
It will be strange to put a link to the samples browser under a sentence says:

> is built from multiple **repositories** in addition to this one:

It might be good to link to the samples browser, but in a separate sentence.